khwilo/questioner-api
# #163068123 Set up the file directory structure

#### What does this PR do?

Sets up the file directory structure for the project.

#### Description of Task to be completed?

- [x] Organize the various project files into their own packages

#### How should this be manually tested?

Once you are on the **ch-file-directory-structure-163068123** branch of this repository, clone the repository to your local machine and see if it conforms to the screenshot provided. On a Linux machine you can install a package known as ***tree*** and execute this command on the terminal:
```bash
$ tree
```

#### Any background context you want to provide?

A guide on project organization can be found here: [Organizing your Flask Application](http://exploreflask.com/en/latest/organizing.html`).

#### What are the relevant pivotal tracker stories?

#163068123

#### Screenshot

![file-directory-structure](https://user-images.githubusercontent.com/5740429/50854813-2dbf1e00-1397-11e9-8644-cbe303fab0e6.png)
